infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationDao.kt

```diff
@@ -731,6 +731,8 @@ class PublicationDao(
                 |layoutBranch=$layoutBranch, publicationId=$publicationId"""
                 .trimMargin()
         }
+        require(layoutBranch == null || layoutBranch == LayoutBranch.main) { """Only main branch supported""" }
+
```

Comment:
Tässä funktiossa (kuten kyllä melkein kaikkialla PublicationDaossa) oli jo aiemminkin oletuksia, joiden takia siitä ei tullut ihan oikeita tuloksia kysyessä suunnitelmatilan muutoksesta, mutta se sattui vähän tuurilla silti toimimaan. Nyt ei enää toimi, joten selkeytin vaatimusta. Käytännössä näitä DAO-metodeja pitää tulevaisuudessa hieman jaotella selkeämmin näin, koska osaa hauista käytetään ainostaan julkaisulokin näyttämisessä (jonka ei tarvitse vielä tukea suunnitelmatilaa), mutta osaa käytetään Ratko-integraatiossa.